homeassistant/components/adguard/sensor.py

```diff
@@ -49,7 +49,7 @@ async def async_setup_entry(
     async_add_entities(sensors, True)
 
 
-class AdGuardHomeSensor(SensorEntity, AdGuardHomeDeviceEntity):
+class AdGuardHomeSensor(AdGuardHomeDeviceEntity, SensorEntity):
```

Comment:
I don't think it is that simple, and quite possibly even incorrect.

I get @MartinHjelmare point he made in a review of <https://github.com/home-assistant/core/pull/48212#discussion_r598766711>, however...

To demonstrate with this use case:

```
Python 3.7.3 (default, Apr 24 2020, 18:51:23)
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> class Entity:
...     pass
...
>>> class SensorEntity:
...     pass
...
>>> class AdGuardHomeEntity(Entity):
...     pass
...
>>> class AdGuardHomeDeviceEntity(AdGuardHomeEntity):
...     pass
...
>>> class AdGuardHomeSensor(AdGuardHomeDeviceEntity, SensorEntity):
...    pass
...
>>> AdGuardHomeSensor.mro()
[<class '__main__.AdGuardHomeSensor'>, <class '__main__.AdGuardHomeDeviceEntity'>, <class '__main__.AdGuardHomeEntity'>, <class '__main__.Entity'>, <class '__main__.SensorEntity'>, <class 'object'>]
```

The result is that `SensorEntity` is now after `Entity`. IMHO, that is incorrect.

Reply:
OK, after I posted this I noticed an error in my train of thought (and example code):

SensorEntity extends from Entity of course... 

`class SensorEntity(Entity):`

🤦  That corrects it

`[<class '__main__.AdGuardHomeSensor'>, <class '__main__.AdGuardHomeDeviceEntity'>, <class '__main__.AdGuardHomeEntity'>, <class '__main__.SensorEntity'>, <class '__main__.Entity'>, <class 'object'>]`